src/test/java/com/amazonaws/services/kinesis/multilang/MultiLangDaemonConfigTest.java

```diff
@@ -67,14 +86,11 @@ public void constructorTest() throws IOException {
 
     @Test
     public void propertyValidation() {
-        String PROPERTIES_NO_EXECUTABLE_NAME =
-                "applicationName = testApp \n" + "streamName = fakeStream \n"
-                        + "AWSCredentialsProvider = DefaultAWSCredentialsProviderChain\n"
-                        + "processingLanguage = malbolge";
+        String PROPERTIES_NO_EXECUTABLE_NAME = "applicationName = testApp \n" + "streamName = fakeStream \n"
```

Comment:
Lowercase variable, i.e. String propertiesNoExecutableName

Reply:
I didn't write this test but, sure I'll change this and the constructorTest so both are lower cased.